src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -139,6 +140,20 @@ public function testGetValueNotModifyObject()
         $this->assertSame(array('Bernhard'), $object->firstName);
     }
 
+    public function testGetValueNotModifyObjectException()
+    {
+        $propertyAccessor = new PropertyAccessor(false, true);
+        $object = new \stdClass();
+        $object->firstName = array('Bernhard');
+
+        try {
+            $propertyAccessor->getValue($object, 'firstName[1]');
```

Comment:
this is missing a `fail()` after wards. Otherwise the test also passed if the exception is not thrown which would be wrong.